concent_api/concent_api/celery.py

```diff
@@ -24,7 +24,15 @@
 for feature_name in settings.CONCENT_FEATURES:
 
     app.conf.imports += tuple(
-        ['{}.tasks'.format(app)
+        [f'{app}.tasks'
          for app in AVAILABLE_CONCENT_FEATURES[feature_name]['required_django_apps']
-         if importlib.util.find_spec('{}.tasks'.format(app)) is not None]
+         if importlib.util.find_spec(f'{app}.tasks') is not None]
     )
+
+app.conf.task_create_missing_queues = True
+app.conf.task_routes = ([
+    ('verifier.tasks.verification_result', {'queue': 'concent'}),
+    ('conductor.tasks.blender_verification_request', {'queue': 'conductor'}),
+    ('verifier.tasks.blender_verification_order', {'queue': 'verifier'}),
+],)
+app.conf.task_default_queue = 'concent'
```

Comment:
This might also be configurable.

Reply:
It really shouldn't. At least not by someone who's setting up Concent on the cluster.

If there are no bugs in the code this default should never come into play. Each task is explicitly routed to the right queue. If we have a bug and someone forgets to route a task I'd like the code to crash so that we notice it an fix it.

Actually, now that I think about it - maybe we could use a name of a non-existent queue. Would it crash then?

Reply:
This seems a good idea.
@cameel ?

Reply:
`task_create_missing_queues` definitely should be disabled. I even stated it in the issue:

> Configure Celery so that new queues are not created automatically.

Think about the use case. When would we ever want a queue to be created automatically? We have a small and constant number of consumer types: concent, conductor and verifier. We know at development time which one should process each task. If won't ever change at runtime. By enabling automatic queue creation the only result is that if you make a typo, your task will go to a new queue that is not being processed by anything. I think it's much, much better if Celery can notify us that we're trying to use a queue that does not exist. If we need a new queue, we'll define it.